### ISSUE TITLE
Itunes 8

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MziRintu/iphonebrowser#55]
```
What steps will reproduce the problem?
1. Open Iphonebrowser (aparently works fine... connect iphone USB)
2. Conectect the iphone to the dock.
3. Iphonebrowser popu up error.
4. Close error kill application.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
- IphoneBrowser 1.6 (updated to 1.7 and same error)
- Windows Xp Professional
- Iphone firmware 2.0.2
- .NETframework 2.0 (updated to 3 and same error)
- downloaded and installed iPhonePCSuite to try (and same error)

Please provide any additional information below.


```

Original issue reported on code.google.com by `javiseg...@gmail.com` on 10 Sep 2008 at 7:50

Attachments:
- [error.jpg](https://storage.googleapis.com/google-code-attachments/iphonebrowser/issue-55/comment-0/error.jpg)

**Comment by GoogleCodeExporter** [comment]
```
I forgot: 

- I'm using Windows XP Pro Service Pack 3, and uninstalled.... the error 
continues.
```

Original comment by `javiseg...@gmail.com` on 10 Sep 2008 at 7:53

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.80
```

Original comment by `NetMage....@gmail.com` on 11 Sep 2008 at 6:09
- Changed state: **Duplicate**
- Added labels: **Priority-Critical**
- Removed labels: **Priority-Medium**
